src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Routing from './components/Routing';
+import './styles/App.css';
+import './styles/Cars.css';
+
+function App() {
```

Comment:
- [ ] For many components and functions, you are using the keyword `function`.  Kindly refactor those functions and components to ES6 arrow functions.

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can make a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/200097223-963602ee-421b-4a03-8853-fd3729866a02.png)


That way, your app will comply with the latest standards 💪